### PR TITLE
Visual updates and fixes for the HIPAA landing page

### PR DESCRIPTION
**Pull request by bwhaley**

### CODE REVIEW

**Review by zackproser — APPROVED** [review]
LGTM

File: pages/landing/hipaa/template/contact/_hero.html [hunk 1]
```diff
@@ -1,8 +1,8 @@
 <div class="container">
   <div class="row">
     <div class="col-xs-12 text-center">
-      <h1>Join the waitlist</h1>
-      <p style="margin-bottom:4px" class="lead">The HIPAA Reference Architecture product is currently in invite-only mode!<br>Please join our waitlist to be notified when it is available.</p>
+      <h1>Request early access</h1>
+      <p style="margin-bottom:4px" class="lead">The HIPAA Reference Architecture product is currently in invite-only mode!<br>Please let us know that you're interested.</p>
```
Comment (brikis98) [inline pages/landing/hipaa/template/contact/_hero.html:5]: Not sure how important consistency is, but in the Heroku landing page, we've updated the text to:

> This product is currently in an invite-only private beta.
> Use this form to request access and we'll notify you when you can get it!

File: pages/landing/hipaa/template/pricing/index.html [hunk 2]
```diff
@@ -18,31 +18,31 @@
     description:
       Early access subscribers will receive a CIS staging environment when invited<br>
       and a fully HIPAA-compliant production infrastructure when it’s ready.
-    price: 3060
-    strike_price: 10000
-    flat_fee: 7425
-    strike_flat_fee: 9900
+    price: "3,060"
+    strike_price: "10,000"
```
Comment (brikis98) [inline pages/landing/hipaa/template/pricing/index.html:22]: It looks like a key change is we no longer show the original price... So the customer has no way of knowing how much of a discount they are getting... That seems to weaken the "discount" proposition, doesn't it?

File: pages/landing/hipaa/template/pricing/index.html [hunk 3]
```diff
@@ -18,31 +18,31 @@
...
+    flat_fee: "7,425"
+    strike_flat_fee: "9,900"
```
Comment (brikis98) [inline pages/landing/hipaa/template/pricing/index.html:24]: Do we need to charge a flat fee at all? It just seems to complicate the decision and doesn't add that much revenue compared to the monthly, recurring fee.

Comment (brikis98) [reply pages/landing/hipaa/template/pricing/index.html:24]: Or perhaps that's one of the big discounts you get as an early subscriber? No "flat fee"?

File: pages/landing/hipaa/template/pricing/index.html [hunk 4]
```diff
@@ -18,31 +18,31 @@
     description:
       Early access subscribers will receive a CIS staging environment when invited<br>
       and a fully HIPAA-compliant production infrastructure when it’s ready.
-    price: 3060
-    strike_price: 10000
-    flat_fee: 7425
-    strike_flat_fee: 9900
+    price: "3,060"
```
Comment (brikis98) [inline pages/landing/hipaa/template/pricing/index.html:21]: That seems rather cheap. A CIS sub goes for $2,040/mo, and customers who need it buy it without too much argument. HIPAA seems like a much bigger, more important, more public facing, audited standard, so I was suggesting ~5x the cost for it, or $10K/mo. Obviously, this should be discounted from that, but should we really discount it by ~70%? That seems to devalue the original sub and the data we'll get from this experiment.

Perhaps we do something like:

- Original price: $9,950/year + $10K up-front
- Discount price: $5,950/year + $0 up-front

File: pages/landing/hipaa/template/pricing/index.html [hunk 5]
```diff
@@ -18,31 +18,31 @@
...
+      - name: HIPAA-compliant sample applications
+        description: World-class sample applications that you can easily extend.
       - name: Infrastructure as Code Library
-        description: Access our entire library to augment and extend your Reference Architecture.
-      - name: Gruntwork Customer Support
-        description: Get help when you need help.
-      - name: Gruntwork Training Libraries
+        description: Access our entire library to augment and extend your reference architecture.
+      - name: Customer support
+        description: Get help when you need it.
```
Comment (brikis98) [inline pages/landing/hipaa/template/pricing/index.html:37]: What support?

File: pages/landing/hipaa/template/pricing/index.html [hunk 6]
```diff
@@ -18,31 +18,31 @@
...
+      - name: Training library
```
Comment (brikis98) [inline pages/landing/hipaa/template/pricing/index.html:38]: Will a customer who has only seen these landing pages have any idea what this is?

File: pages/landing/hipaa/template/pricing/index.html (hunk 2 above)
Comment (bwhaley) [reply pages/landing/hipaa/template/pricing/index.html:22]: I do not have a strong view on this. This change is in response to [Josh's comments on Slack](https://gruntwork-io.slack.com/archives/C0275GS2FKM/p1626909175224300?thread_ts=1626879024.181400&cid=C0275GS2FKM). If we want to keep the crossed out pricing that's fine.

File: pages/landing/hipaa/template/pricing/index.html (hunk 4 above)
Comment (bwhaley) [reply pages/landing/hipaa/template/pricing/index.html]: Updated in 83985751.

File: pages/landing/hipaa/template/pricing/index.html (hunk 5 above)
Comment (bwhaley) [reply pages/landing/hipaa/template/pricing/index.html]: Updated wording in 83985751.

File: pages/landing/hipaa/template/pricing/index.html (hunk 6 above)
Comment (bwhaley) [reply pages/landing/hipaa/template/pricing/index.html]: Updated in 83985751.

File: pages/landing/hipaa/template/pricing/index.html (hunk 3 above)
Comment (bwhaley) [reply pages/landing/hipaa/template/pricing/index.html:24]: Updated in 83985751.

File: pages/landing/hipaa/template/pricing/index.html (hunk 2 above)
Comment (bwhaley) [reply pages/landing/hipaa/template/pricing/index.html]: Updated in 83985751.

File: pages/landing/hipaa/template/contact/_hero.html (hunk 1 above)
Comment (bwhaley) [reply pages/landing/hipaa/template/contact/_hero.html]: Updated in 83985751.

**Comment by bwhaley** [comment]
Alrighty, hopefully this is ready to merge.

**Review by zackproser — APPROVED** [review]
LGTM

**Comment by bwhaley** [comment]
Thx. Will go ahead and merge this. If there are more follow ups we can submit them as individual issues.

**Pull request merged**